server/src/main/java/com/cloud/configuration/Config.java

```diff
@@ -134,6 +134,22 @@ public enum Config {
             "0.75",
             "Percentage (as a value between 0 and 1) of local storage utilization above which alerts will be sent about low local storage available.",
             null),
+    BackupStorageCapacityThreshold(
```

Comment:
Would it be better to define them as ConfigKey in the respective manager/service?